components/squad/squad_row.lua

```diff
@@ -118,10 +138,12 @@ function SquadRow:newteam(args)
 	end
 
 	self.content:node(cell)
+
 	return self
 end
 
-function SquadRow:create()
+function SquadRow:create(id)
+	mw.ext.LiquipediaDB.lpdb_squadplayer(id, self.lpdbData)
 	return self.content
```

Comment:
```suggestion
function SquadRow:create(id)
	self.lpdbData = self:addToLpdb()
	mw.ext.LiquipediaDB.lpdb_squadplayer(id, self.lpdbData)
	return self.content
```
so we can add custom stuff for extradata (e.g. factions and such)
(alternative ways to achieve that would of course be okay too^^)

Reply:
Done